youToBe.css

```diff
@@ -0,0 +1,66 @@
+body {
+    background-image: url(https://images.saymedia-content.com/.image/t_share/MTc4NzM1OTc4MzE0MzQzOTM1/how-to-create-cool-website-backgrounds-the-ultimate-guide.png);
+    background-position: center;
+    background-size: cover;
+    background-repeat: round;
+}
+
+#homeBody {
+    width: 900px;
```

Comment:
Great using of styles so far! Just remember is a good practice to create mobile first and responsive websites, I suggest using relative space instead of fixed sizes for your content:

`#my-id {
width: 100%;
}`